### PR TITLE
2024 Organize current folder structure

### PR DESCRIPTION
**Pull request by FerdausPolok**
In this PR, I've

- Organized the current file folders.
- Created initial `readme`s where necessary.

Will request original authors to update there readme's if possible. Will try to mention them below. Thanks. 😀

### CODE REVIEW

File: apis/apiClient.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## About ApiClint
+
+Todo from [Tanvir Imam Mitul
+](https://github.com/tanvirim) if he's available 😛
```
Comment (FerdausPolok) [inline apis/apiClient.md:4]: Yo @tanvirim bro👋
Interested in Hacktoberfest this year? You can start by making a PR to update this README. That'll be your first PR, then you can do 3 more as per your wish. 

Thanks!

File: applications-code-snippets/python-admission-system-models/readme.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## About Admission System Models
+
+Todo from @[PollenLight](https://github.com/PollenLight) Bhai if he's available 😛
```
Comment (FerdausPolok) [inline applications-code-snippets/python-admission-system-models/readme.md:3]: Hello @PollenLight Bhai👋

Interested in Hacktoberfest this year? You can start by making a PR to update this README. That'll be your first PR, then you can do 3 more as per your wish.

Thanks!

File: applications-code-snippets/python-hashkey-generator/readme.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+## About Admission System Models
+
+Todo from @[Mubtasim Fuad](https://github.com/mubtasimfuad) if he's available 😛
```
Comment (FerdausPolok) [inline applications-code-snippets/python-hashkey-generator/readme.md:3]: Hello @mubtasimfuad Bhai👋

Interested in Hacktoberfest this year? You can start by making a PR to update this README. That'll be your first PR, then you can do 3 more as per your wish.

Thanks!

**Review by aktrisha — APPROVED** [review]
LGTM

**Pull request merged**